packages/wdio-cli/src/config.js

```diff
@@ -14,7 +14,8 @@ export const SUPPORTED_REPORTER = [
     ' json - https://www.npmjs.com/package/wdio-json-reporter',
     ' concise - https://www.npmjs.com/package/wdio-concise-reporter',
     ' testrail - https://www.npmjs.com/package/wdio-testrail-reporter',
-    ' mochawesome - https://www.npmjs.com/package/wdio-mochawesome-reporter'
+    ' mochawesome - https://www.npmjs.com/package/wdio-mochawesome-reporter',
+    ' timeline - https://www.npmjs.com/package/wdio-mochawesome-reporter'
```

Comment:
should be
```
' timeline - https://www.npmjs.com/package/wdio-timeline-reporter'
```

Reply:
🤦‍♂️ had to delete previous commit and re-committed as my git user was not set up. Will make change now